CHANGELOG.md

```diff
@@ -3,18 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## Before Initial Release
+## 0.4.2
 
-- feat: initial commit 🎉
+- First stable deployment.
```

Comment:
Is this comment needed, shouldn't the version number represent stable status?

Reply:
maybe the description is the problem? what i wanted to highlight is that this version was the first release...
maybe "First release to pub.dev."?

Reply:
updated